### PR TITLE
Shovel Knight 4.1 (EUR) more cheats added

### PR DESCRIPTION
**Pull request by subworx**

### CODE REVIEW

**Comment by JourneyOver** [comment]
Please remove things like:
```[++(Random name)++]```, ```[--]```

as they are not supported by lumas cheat engine.

**Comment by subworx** [comment]
Thank you for the info, creating a new request with proper formatting

**Comment by subworx** [comment]
oh, github automatically updated the pull request with the folder removing commits. weird piece of software...

File: 3ds/0004000E0017C900.txt [hunk 1]
```diff
@@ -1,24 +1,44 @@
...
 
-[99999 Money (Game v4.1)]
+[Shovel K. 99,999 Money (v4.1)]
 D3000000 30000000
 6109F594 00000000
 B109F594 00000000
 000006F4 0001869F
 D2000000 00000000
 
+[++Plague Knight (v4.1)++]
```
Comment (JourneyOver) [inline 3ds/0004000E0017C900.txt:22]: You're missing this last one, remove this one and I can merge this in.

**Comment by subworx** [comment]
seems like i am a tiny bit stupid :/ fixed now. 
**Thank you so much for your patience.**
planned on working on new codes for some games and will make sure that future additions are proper in the first place.

**Pull request merged**